README.md

```diff
@@ -26,32 +26,29 @@ This implementation shows you how to do the following:
+|us-east-2 (Ohio)          | [<img src="./notebooks/images/cloudformation-launch-stack.png">](https://console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/new?stackName=sm-fs-streaming-agg-stack&templateURL=https://s3.amazonaws.com/sagemaker-feature-store-streaming-aggregation-us-east-2/artifacts/latest/sagemaker-featurestore-template.yaml) |
+|us-west-1 (N. California) | [<img src="./notebooks/images/cloudformation-launch-stack.png">](https://console.aws.amazon.com/cloudformation/home?region=us-west-1#/stacks/new?stackName=sm-fs-streaming-agg-stack&templateURL=https://s3.amazonaws.com/sagemaker-feature-store-streaming-aggregation-us-west-1/artifacts/latest/sagemaker-featurestore-template.yaml) |
+|eu-west-1 (Dublin)        | [<img src="./notebooks/images/cloudformation-launch-stack.png">](https://console.aws.amazon.com/cloudformation/home?region=eu-west-1#/stacks/new?stackName=sm-fs-streaming-agg-stack&templateURL=https://s3.amazonaws.com/sagemaker-feature-store-streaming-aggregation-eu-west-1/artifacts/latest/sagemaker-featurestore-template.yaml) |
+|ap-northeast-1 (Tokyo)    | [<img src="./notebooks/images/cloudformation-launch-stack.png">](https://console.aws.amazon.com/cloudformation/home?region=ap-northeast-1#/stacks/new?stackName=sm-fs-streaming-agg-stack&templateURL=https://s3.amazonaws.com/sagemaker-feature-store-streaming-aggregation-ap-northeast-1/artifacts/latest/sagemaker-featurestore-template.yaml) |
 
-1. [Launch](https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/create/template?stackName=sm-fs-streaming-agg-stack&templateURL=https://aws-ml-blog.s3.us-east-1.amazonaws.com/artifacts/Using-streaming-ingestion-with-Amazon-SageMaker-Feature-Store/sagemaker-featurestore-template.yaml) the AWS CloudFormation stack in us-east-1. 
-To deploy the stack in other regions, [follow these instructions](./create_stack_in_other_regions.md).
+To deploy the stack in other regions, you can [follow these instructions](./create_stack_in_other_regions.md). Please log an issue in this repo if you would like additional regions officially supported.
 
-  - You will need to give your CloudFormation stack a name. We suggest you enter the preferred name -> `sm-fs-streaming-agg-stack` (note, if you use a different Stack name, you will need to update the cell at the top of notebook named `1_setup.ipynb` to match it)
-  - During CloudFormation configuration, you can easily select between several ML Instance Types, used by SageMaker to deploy the Notebook, using the selector as shown here:
-<img src="./notebooks/images/CFN-Param-InstanceType-select.png" />
+2. Click 'Next' for 'Specify template', 'Specify stack details', and 'Configure stack options'. On the 'Review' step, check the box that says 'I acknowledge that AWS CloudFormation might create IAM resources with custom names.' and then click 'Create Stack'. 
```

Comment:
You have omitted the "hint" about setting Stack name, which means that the user a) will not know what to type in the name, and b) may not update the corresponding location in the notebook. Shouldn't we continue to provide this hint to the end user?

Reply:
All of the `Launch Stack` links will pre-fill the Stack Name parameter with the preferred name so users no longer need to worry about this. Notebook 1 includes a comment in the cell that saves the stack name as a variable to change this string if you opted for a different stack name in the CFN deployment.